doc/source/dev/development_environment.rst

```diff
@@ -64,6 +64,12 @@ This builds NumPy first, so the first time it may take a few minutes.  If
 you specify ``-n``, the tests are run against the version of NumPy (if
 any) found on current PYTHONPATH.
 
+.. note::
+
+    If the above commands return a RuntimeError with the message "Cannot parse version 0+untagged.xxxxx,"
+    it indicates that you are currently developing on a fork that lacks release tags.
+    To resolve this issue, execute the following command in your ``main`` branch: ``git pull upstream main --tags`` 
+
```

Comment:
```suggestion
.. note::

    If the above commands result in ``RuntimeError: Cannot parse version...``,
    run ``git pull upstream main --tags``.

```

Just a wording suggestion to try to distill down to the critical info and conform to an 80-ish character line limit. Neither of these are blockers.

Reply:
Thanks for checking it out! I applied most of your suggestions.